lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
 
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
+  hash = {}
+  results = []
+
+  string.each_char do |char|
+    if hash[char] == nil
+      hash[char]== 0
```

Comment:
Should this be 0 or 1?  Also I don't think you mean to use two equal signs here.